KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaCuePointsManager.java

```diff
@@ -107,7 +107,7 @@ public void load(final List<String> entryIds, boolean periodic) {
 			
 			try{
 				while(true){
-					KalturaCuePointListResponse CuePointsList = client.getCuePointService().list(filter , pager);
+					KalturaCuePointListResponse CuePointsList = getClient().getCuePointService().list(filter , pager);
```

Comment:
No need to create new instance of the client inside the loop, create it once before the loop.